internal/pkg/cli/dialog/use_template.go

```diff
@@ -64,125 +64,113 @@ func (d *useTmplDialog) ask(inputs input.StepsGroups) (useTemplate.Options, erro
 	d.out.TargetBranch = targetBranch.BranchKey
 
 	// User inputs
-	if err := d.askInputs(inputs); err != nil {
+	if err := d.askInputs(stepsGroups.ToExtended()); err != nil {
 		return d.out, err
 	}
 
 	return d.out, nil
 }
 
-// addInputValue from CLI dialog or inputs file.
-func (d *useTmplDialog) addInputValue(value interface{}, inputDef input.Input, validate bool) error {
-	// Convert
-	value, err := inputDef.Type.ParseValue(value)
-	if err != nil {
-		return fmt.Errorf("invalid template input: %w", err)
-	}
+func (d *useTmplDialog) askInputs(stepsGroups input.StepsGroupsExt) error {
+	return stepsGroups.VisitInputs(func(group *input.StepsGroupExt, step *input.StepExt, inputDef input.Input) error {
+		// Print info about group and select steps
+		if !group.Announced {
+			if err := d.announceGroup(group); err != nil {
```

Comment:
`announceGroup` zobrazi selectBox, ak je to potrebne a nastavi kazdemu `step.Show`, a to sa pouzije dalej.